code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -168,6 +186,30 @@ GLOBAL_LIST_EMPTY(apostles)
 			if(length(possible_locs) > 1)
 				possible_locs -= T
 
+/mob/living/simple_animal/hostile/abnormality/white_night/proc/DesignateApostles()
+	for(var/mob/living/simple_animal/hostile/abnormality/A in view(vision_range, src))
+		if(A == src)
+			continue
+		if(A.status_flags & GODMODE)
+			continue
+		if(A.threat_level >= WAW_LEVEL)
+			A.faction += "apostle"
+			apostles += A
+			A.name += " the [apostles.len]\th Apostle"
```

Comment:
Why the backslash?

Reply:
Built-in string that will automatically do the appropriate "nth", for example the first one would be the "1st" while the third would be the "3rd"